test/controllers/cars_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class CarsControllerTest < ActionDispatch::IntegrationTest
+  test "should get brand:string" do
+    get cars_brand:string_url
```

Comment:
I don't know if I've seen something like `cars_brand:string_url` before.  Curious to know what it does.